src/routes/api/api-ping-post.ts

```diff
@@ -16,6 +19,34 @@ export const ApiPingPost = async (req: Request, res: Response): Promise<void> =>
 		return;
 	}
 
+	try {
+		const output = await runCurl({
+			fullUrl: data.url,
+			method: "GET",
+			authorization: ""
+		});
+		logCurlOutputInChunks(output, req.log);
```

Comment:
I am hoping this will give us output on `ssl verification: OK` or `ssl verification: fail` on the `curl` command.

Reply:
btw do we use outbound proxy for curl commands? I think this customer is using ip whitelisting, which means it will timeout if not

Reply:
Good point. I don't think the curl use outbound. Let me see how to do it

Reply:
fixed.